appendices.tex

```diff
@@ -370,6 +370,146 @@ \chapter{Listing of application roles}
+% 	\item[cache\_templates] Determines if HTML templates used for the LedgerSMB UI will be stored
+% 	in precompiled form for improved performance. The value 0 (zero) means no caching; the value
+% 	1 means caching in the directory \$(tempdir)/lsmb\_templates; systems running multiple
+% 	LedgerSMB instances with customized UIs should use separate values for {\tt tempdir} for that
+% 	reason
+% 	\item[DBI\_TRACE] @@@TODO
+% 	\item[pathsep] @@@TODO
+% 	\item[latex] Identifies if the system has \LaTeX available: 0 means no \LaTeX, 1 means \LaTeX supported
+% 	\item[check\_max\_invoices] @@@TODO
+% 	\item[max\_post\_size] States the maximum size in bytes of the body in case of a POST request; this
+% 	setting reduces the impact of DoS requests, if the system is under attack. The setting also limits the
+% 	maximum size of attachments (see \secref{sec-workflows-quotations-file-attachments}) that can be uploaded
+% 	\item[decimal\_places] @@@TODO
+% 	\item[cookie\_name] @@@TODO
+% 	\item[no\_db\_str] @@@TODO
+	
+\end{description}
+
+\subsubsection{[environment] Section}
+\label{subsubsec-global-config-ledgersmb-conf-environment}
+
+	Sets shell environment variables when starting up.
+
+\begin{description}[style=nextline]
+	\item[PATH] If the server can't find applications, append to the path. For example:
+	\texttt{PATH=/bin:/usr/bin:/usr/local/bin:/usr/local/pgsql/bin}
+	\item [PERL5LIB] On some operating systems like macOS the perl path may need to be set. For example, on macOS using Fink's Perl libs it might be set like:
+	\texttt{PERL5LIB=/sw/lib/perl5}
+\end{description}
+
+
+\subsubsection{[paths] Section}
+\label{subsubsec-global-config-ledgersmb-conf-paths}
+
+\begin{description}[style=nextline]
+	\item [templates] Templates base directory. Typically set to \texttt{templates}.
+	\item [images] Images base directory. Typically set to \texttt{UI/images}.
+	\item [localepath] Path to locale files. Defaults to \texttt{locale/po}.
+	\item [templates\_cache] Location where compiled templates are stored. When relative, appended to the directory specified by \texttt{File::Spec->tmpdir()}. 
+	
+	Defaults to \texttt{lsmb\_templates}.
+	\item [workflows] Directory where workflow files are stored. Defaults to \texttt{workflows}
+	\item [custom\_workflows] Directory where custom workflow files are stored. Custom workflows are used to override behavior of the default workflows by providing actions, conditions, etc. with the same name and type or by providing workflows of the same type with additional states and actions. Defaults to \texttt{custom\_workflows}.
+	
+% Items from the old book
+%	\item [spool] @@@TODO
+%	\item [userspath] @@@TODO
+%	\item [memberfile] @@@TODO
+\end{description}
+
+\subsubsection{[programs] Section}
+\label{subsubsec-global-config-ledgersmb-conf-programs}
+
+Setup the various helper programs that can be used with LedgerSMB.
+
+% From the old book.
+%\begin{description}[style=nextline]
+%	\item [gzip] @@@TODO

```

Comment:
Doesn't look like it's used in 1.10 anymore.

Reply:
Removed.